phantazm-zombies-mapeditor/src/main/resources/mapeditor.mixins.json

```diff
@@ -0,0 +1,13 @@
+{
+  "required": true,
+  "minVersion": "0.8",
+  "package": "com.github.phantazmnetwork.zombies.mapeditor.mixin",
+  "compatibilityLevel": "JAVA_8",
```

Comment:
not that I have any idea how this works but wouldn't this be 17

Reply:
not sure, but it works and i'm afraid to touch it